spec/researches/metaDescriptionKeywordSpec.js

```diff
@@ -1,40 +1,45 @@
 import metaDescriptionKeyword from "../../src/researches/metaDescriptionKeyword.js";
 import Paper from "../../src/values/Paper.js";
+import Factory from "../helpers/factory.js";
+
+const mockResearcherWord = Factory.buildMockResearcher( { keyphraseForms: [ [ "word", "words" ] ] } );
+const mockResearcherWordUmlaut = Factory.buildMockResearcher( { keyphraseForms: [ [ "keywörd", "keywörds" ] ] } );
+const mockResearcherWordDollarSign = Factory.buildMockResearcher( { keyphraseForms: [ [ "$keyword", "$keywords" ] ] } );
```

Comment:
The assessment needs to take synonyms as well, so please add synonymForms and respective tests for them.